assets/css/components/general.css

```diff
@@ -0,0 +1,15 @@
+/* csslint ignore:start */
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+/* csslint ignore:end */
```

Comment:
I could be wrong but I think you only need to declare the `@tailwind` statements once, now we have them in both `navbar.css` and `general.css`. Can you try moving them to `app.css` and see if that works?